docs/rooosss/scripts/pokemons.js

```diff
@@ -0,0 +1,95 @@
+const cardsList = document.querySelector('[data-type="cards-list"]');
+const formFilters = document.querySelector('[data-type="filters"]');
+const loading = document.querySelector('[data-type="loading"]');
+const bigPokemonCheckbox = formFilters.elements.big;
+const smallPokemonCheckbox = formFilters.elements.small;
+const baseUrl = 'https://my-json-server.typicode.com/electrovladyslav/pokemon-json-server/pokemons';
+let pokemons = [];
+
+function createCardElement(pokemonsList) {
+    const cardItem = document.querySelector('[data-type="card-template"]');
+    const cardContent = cardItem.content;
+    const cardName = cardContent.querySelector('[data-card-name]');
+    const cardImage = cardContent.querySelector('[data-card-image]');
+    const cardHeight = cardContent.querySelector('[data-card-height]');
+    const cardWeight = cardContent.querySelector('[data-card-weight]');
+    const cardNumber = cardContent.querySelector('[data-card-number]');
+
+    cardImage.src = pokemonsList.ThumbnailImage;
+    cardName.textContent = pokemonsList.name;
+    cardHeight.textContent = pokemonsList.height;
+    cardWeight.textContent = pokemonsList.weight;
+    cardNumber.textContent = pokemonsList.number;
+    return cardContent.cloneNode(true);
+}
+
+function renderCards(container, pokemonsList) {
+    container.innerHTML = '';
+    const fragment = new DocumentFragment();
+    pokemonsList.forEach((pokemon) => {
+        const card = createCardElement(pokemon);
+        fragment.appendChild(card);
+    });
+    container.appendChild(fragment);
+}
+
+function filterPokemons(pokemonsList) {
+    const isSmallSizeFilterChecked = formFilters.elements.small.checked;
+    const isBigSizeFilterChecked = formFilters.elements.big.checked;
+    const filter = [];
+    pokemonsList.forEach((card) => {
+        if (isBigSizeFilterChecked && card.height >= 100) {
```

Comment:
I suggest using some other values, lest say 20'' and 50``. That would add a visual representation of clicking `big` checkbox.
And you can add it as a description to checkboxes texts: `Big(>50'')`

Reply:
you can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter for the filtering propose
Also 
> При обраних чекбоксах Big та Small  - покемони нижче 50' та вище 100'

Now when both checkboxes are checked a user can see only small pokemons

Reply:
And you can declare the filters values (20 and 50) as constants on the top of the script with some decent name. [Magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) - is an antipattern